src/lib.rs

```diff
@@ -50,7 +50,8 @@
 // Clippy does not have `broken_intra_doc_links` as a known lint.
 #![allow(unknown_lints)]
 #![deny(broken_intra_doc_links)]
-#![deny(missing_debug_implementations)]
+// XXX: This needs to go if we are moving to no_std compatibility.
+// #![deny(missing_debug_implementations)]
```

Comment:
Not sure why. Can't we use the `core` one?

Reply:
It failed the check for me, so I don't think so..

Reply:
You know what? Let's remove this lint for now. Since there is quite a lot of structs that we don't need to impl Debug for and that will affect the compile time for absolutely no reason.

Reply:
Sure!